main/src/library/Reflect.flix

```diff
@@ -0,0 +1,18 @@
+namespace Reflect {
+
+    @Unsafe
```

Comment:
No comment on what this does? :)

https://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5